support-frontend/assets/pages/[countryGroupId]/components/thankyou.tsx

```diff
@@ -215,7 +215,10 @@ export function ThankYouComponent({
 	): ThankYouModuleType[] => (condition ? [moduleType] : []);
 
 	const thankYouModules: ThankYouModuleType[] = [
-		...maybeThankYouModule(isNewAccount, 'signUp'), // Create your Guardian account
+		...maybeThankYouModule(
```

Comment:
Why are we hiding the sign up component?

Reply:
Was shown like this in the requirements [document](https://docs.google.com/document/d/176biaQc9eWQnSa1H1pDt_z_o_quRWTxOETGpCtaU1Qc/edit?usp=sharing), presume reduce the clutter of thank-you components?

![image](https://github.com/user-attachments/assets/70567fd0-30b5-4fe5-9628-50680aef2fe2)

Reply:
That could possibly just be the signed in version

Reply:
![image](https://github.com/user-attachments/assets/46a94909-793f-4eca-bef4-e8aaf2ad7260)

The figma still includes the other versions